### PR TITLE
Templates: Properly Access getter/setter for is-Variables on JVM

### PR DESCRIPTION
**Pull request by NWuensche**

### CODE REVIEW

File: korlibs-template/test/korlibs/template/TemplateTest.kt [hunk 1]
```diff
@@ -51,6 +51,22 @@ class TemplateTest : BaseTest() {
         assertEquals("soywizsoywiz", KorteTemplate("{{name}}{{ name }}")("name" to "soywiz"))
     }
 
+    @Test
+    fun testSimpleIs() = suspendTest {
```
Comment (soywiz) [inline korlibs-template/test/korlibs/template/TemplateTest.kt:55]: This test should be moved here, so it is only executed on the JVM: `https://github.com/korlibs/korlibs/blob/main/korlibs-template/test%40jvm/korlibs/template/TemplateJvmTest.kt`

**Comment by NWuensche** [comment]
Done, thank you!

File: korlibs-template/.idea/workspace.xml
Comment (soywiz) [inline korlibs-template/.idea/workspace.xml:1]: I guess this was added by mistake. Can you delete this file from the PR?

**Comment by soywiz** [comment]
In the end I have deleted the file myself so I could merge it already. Thanks!

**Pull request merged**